rfc/12/README.md

```diff
@@ -81,4 +81,4 @@ There are 2 options for health checks integration into Pacemaker:
 
 ## See also
 
-* [Original list of LDR1 HA health checks. The spreadsheet has a lot of useful information](https://docs.google.com/spreadsheets/d/1xASlPnlFx1OmhKttbgOweHbmfbXMI6n3Pmp2TSO8ou8/edit#gid=1305517710)
+* [Original list of LDR1 HA health checks. The spreadsheet has a lot of useful information](https://seagatetechnology.sharepoint.com/:x:/r/sites/gteamdrv1/tdrive1224/LDR%20R1/HA/HA%20requirements%202020-01-14%20.xlsx?d=wdf76a255f0b84af5a771744ef00440d5&csf=1&web=1&e=gKXtZM)
```

Comment:
Can we try out techniques of embedding excel and ppt documents in confluence page, as shown in these videos?
https://www.youtube.com/watch?v=MhwKcD-Z83w
https://www.youtube.com/watch?v=1Ubm0DlnbVU

If it works then we can have our excel and ppt documents in confluence pages.
We can create a confluence page with title, what the sheet or ppt doc has, and embed the document in that confluence page.
Hopefully that will work.

Reply:
@suvratjoshi  As per yesterday's tech discussion, It discussed that we only create, confluence pages for design documents/Integration documents.

We don't need to convert these requirement-related documents. So as per discussion, I added it into this shared location https://seagatetechnology.sharepoint.com/sites/gteamdrv1/tdrive1224/LDR%20R1/Forms/AllItems.aspx?id=%2Fsites%2Fgteamdrv1%2Ftdrive1224%2FLDR%20R1%2FHA&viewid=c3aa8d53%2D081b%2D4b29%2D82ee%2D76108af397de
cc @mssawant @pavankrishnat

Reply:
Thats ok.
Then we need to have method to share excel/ppt docs with outside world.

Reply:
Yes checking for that.

Reply:
Other way to make docs accessible is by creating a directory named something like "ref_docs" and put excel and ppt like docs in that dir and link to those.